operator/src/main/java/com/github/streamshub/console/LeaderConfiguration.java

```diff
@@ -7,6 +7,6 @@
 @ApplicationScoped
 public class LeaderConfiguration extends LeaderElectionConfiguration {
     public LeaderConfiguration() {
-        super("console-operator-lease");
+        super("streamshub-console-operator-lease");
```

Comment:
Not sure if this is needed or not, but i'll leave it up to review.

Reply:
I think this should be fine